src/NzbDrone.Common/Http/HttpRequestBuilder.cs

```diff
@@ -253,6 +262,19 @@ public virtual HttpRequestBuilder SetHeader(string name, string value)
             return this;
         }
 
+        public virtual HttpRequestBuilder AddPrefixQueryParam(string key, object value, bool replace = false)
+        {
+            if (replace)
+            {
+                QueryParams.RemoveAll(v => v.Key == key);
+                SuffixQueryParams.RemoveAll(v => v.Key == key);
+            }
+
+            QueryParams.Insert(0, new KeyValuePair<string, string>(key, value.ToString()));
```

Comment:
Having less important stuff at the end (apikey, etc) makes it easier to read in log files. That's it.

Having suffix separately saves me from having an integer somewhere keeping track of the insertion point.